portfolio/src/main/webapp/style.css

```diff
@@ -52,8 +64,60 @@ footer {
   background: black;
 }
 
+section {
+  animation: fadeIn 1s ease-in-out;
+}
+
+.section-content {
+  display: none;
+  padding: 6rem;
+}
```

Comment:
This is a class for all the content which is not the main one and will be rendered with the commands or navbar. It substitutes the ids @livvielin told me just for the styling.

Reply:
Awesome!